### PR TITLE
fix: pixi global required args

### PR DESCRIPTION
**Pull request by nichmor**
https://github.com/clap-rs/clap/issues/4507#issuecomment-1372250231
`num_args = 1.. ` don't make packages a mandatory argument.
This PR aims to fix this, and some other stylistic usage of report instead of computing println message

### CODE REVIEW

File: src/cli/global/update.rs [hunk 1]
```diff
@@ -66,11 +65,8 @@ pub async fn execute(args: Args) -> miette::Result<()> {
 
         // Remove outdated binaries
         state_changes |= project.prune_exposed(env_name).await?;
-        eprintln!(
-            "{}Updated environment: {}.",
-            console::style(console::Emoji("✔ ", "")).green(),
-            env_name.fancy_display()
-        );
+
+        state_changes.report();
```
Comment (Hofer-Julian) [inline src/cli/global/update.rs:69]: The report already happens on line 99. I think this should be reverted

**Review by Hofer-Julian — COMMENTED** [review]
One small change, apart from that it looks good to me

File: src/cli/global/update.rs (hunk 1 above)
Comment (nichmor) [reply src/cli/global/update.rs:69]: that report is applied on newer StateChanges object ( it is init on line 82 )

**Review by Hofer-Julian — APPROVED** [review]
Awesome!

**Pull request merged**